plugins/accounts/build.gradle.kts

```diff
@@ -43,18 +43,23 @@ android {
     }
 
     buildFeatures {
+        compose = true
         viewBinding = true
     }
-    namespace = "com.redmadrobot.account_plugin"
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = androidx.versions.compose.compiler.get()
+    }
```

Comment:
Also, not related changes

Reply:
See https://github.com/RedMadRobot/debug-panel-android/pull/16#discussion_r1580774193